### PR TITLE
feat(addScript): disable `'use strict'` to allow setting globals

### PR DESCRIPTION
**Pull request by timpur**

### CODE REVIEW

**Comment by timpur** [comment]
@d3viant0ne @bebraw @SpaceK33z @TheLarkInn

**Comment by antonyoneill** [comment]
Hey @bebraw, @TheLarkInn @d3viant0ne, @evilebottnawi 

What is it going to take to get some love for this repository?

This PR should be considered for merge, as should the documentation one #45.

**Comment by michael-ciniawsky** [comment]
This may need to be an option, e.g `options.strict = true || false`, instead if being the default. Do we have an overview of potential security implications?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "script-loader",
-  "version": "0.7.2",
+  "version": "0.7.3",
```
Comment (michael-ciniawsky) [inline package.json]: Please revert this as it is the maintainers responsibility to do this

Comment (timpur) [reply package.json]: sorry, my bad, will do

**Comment by michael-ciniawsky** [comment]
@timpur What's the status here? @timpur Will you continue this PR (no pressure, but please indicated if you're still interested in this)?

**Comment by timpur** [comment]
@michael-ciniawsky, i am just not sure how to do the option thing that you requested tbh...

**Comment by antonyoneill** [comment]
Hey @timpur take a look at how @jiaweihli has done similar in this PR https://github.com/webpack-contrib/script-loader/pull/31/files#diff-168726dbe96b3ce427e7fedce31bb0bc

**Comment by michael-ciniawsky** [comment]
Yep, e.g https://github.com/webpack-contrib/script-loader/pull/31/files#diff-168726dbe96b3ce427e7fedce31bb0bc is a good starting point, also feel free to ask questions and push the code if you're stuck somewhere :)

**Comment by antonyoneill** [comment]
Hey guys, let me know if there's anything I can do to get this shepherded into the project.

**Comment by timpur** [comment]
Some one want to take over this PR ?

**Comment by FilippoPolo** [comment]
Hi all,
I've sent @timpur a PR that adds an option to his logic, so that existing behavior is preserved for current users. You would add ?stripUseStrict to trigger the feature.

**Comment by timpur** [comment]
update the pr and added documentation, changed the option to` useStrict: false` to disabled use strict, enabled by default. Can change interface if you dont like.

**Comment by gfortaine** [comment]
Hello @michael-ciniawsky

Any plan to merge this PR any time soon, please ? Indeed, as others mentioned, it has proven very useful in some cases https://github.com/webpack-contrib/script-loader/pull/51

**Comment by mistic** [comment]
@evilebottnawi merged

**Pull request merged**